demisto_sdk/commands/common/tools.py

```diff
@@ -289,20 +291,24 @@ def get_mp_tag_parser():
     return MARKETPLACE_TAG_PARSER
 
 
-def get_yml_paths_in_dir(project_dir: str, error_msg: str = "") -> Tuple[list, str]:
+def get_yml_paths_in_dir(project_dir: str | Path, error_msg: str = "") -> Tuple[list, str]:
```

Comment:
This will break other functions passing a string for this method (and I prefer not to change non-related functions in this PR, to not break unrelated-stuff)